src/Controller/AbstractConfigController.php

```diff
@@ -114,7 +114,8 @@ protected function getHeaderType($headers, $header)
         $accept = strtolower(trim($accept));
         switch ($accept) {
             case 'application/json':
-            case 'application/vnd.laminascampus.v1.config+json':
```

Comment:
Agreed with @froschdesign here - we should keep this, and add a `@todo` to remove it for next major.

Reply:
Added back, so now we have three options, but in next major we want to keep only one: `application/vnd.laminas-api-tools.v1.config+json`.